masterlist.yaml

```diff
@@ -715,8 +715,14 @@ plugins:
     after:
       - 'Realistic Lighting Overhaul - Major City Interiors.esp'
   - name: 'BetterQuestObjectives.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/159/']
+    # Even Better Quest Objectives
```

Comment:
Could the global priority for this mod be set to (-1) as it just replaces quest entry names & objectives. Any mods it does conflict with it has a patch included or is easily patched. It doesn't have a patch included for Amazing follower tweaks or Immersive AFT which it will break if it overwrites. Or I could add a load after rule to those mods.